model-optimizer/mo/front/mxnet/extractors/slice_axis.py

```diff
@@ -29,3 +32,40 @@ def slice_axis_ext(attrs):
         'infer': mxnet_slice_axis_infer
     }
     return node_attrs
+
+
+def mxnet_slice_axis_infer(node):
+    in_shape = node.in_node(0).shape
+    node.axis = get_canonical_axis_index(in_shape, node.axis)
+    slice_axis = node.axis
+
+    new_shape = np.array(in_shape, dtype=np.int64)
```

Comment:
`new_shape` = int64_array(in_shape)

Reply:
I missed your comment. Left as it was before. I just cutpasted the old code, without changes. The current code works. I assume that priority of your comment is `nit:` and it's rather a wish than necessity.